### DISCUSSION TITLE
Will classic mod be ranked?

### DISCUSSION THREAD

**Discussion by LunaticaUWU** [ppy/osu#25350] [Q&A] [answered]
Hey. Is classic mod going to give pp after the release of osu!lazer? If yes, will the pp system work the same as current stable (with some balancing, to not be overpowered over normal lazer), or will it be fully dependent on new lazer system?

**Comment by peppy** [comment, marked as answer]
Likely yes.

**Reply by LunaticaUWU** [reply]
(edited)

**Reply by LunaticaUWU** [reply]
And the 2nd question?

**Reply by Givikap120** [reply]
it's more likely to lazer have advantage over stable than vice versa
